lib/apiservers/engine/backends/archive.go

```diff
@@ -190,9 +198,18 @@ func (c *Container) ContainerStatPath(name string, path string) (stat *types.Con
 		return nil, NotFoundError(name)
 	}
 
+	path = filepath.Clean(path)
 	mounts := mountsFromContainer(vc)
 	mounts = append(mounts, types.MountPoint{Destination: "/"})
 
+	// handle the special case of targeting a volume mount point before it exists.
+	// this will be important for non started container cp, will also be important
+	// to certain behaviors for diff on a non started container.
+	if stat, succeed := tryFakeStatPath(mounts, path); succeed {
```

Comment:
The benefit of faking statpath is that we can avoid making calls to portlayer and mount vmdk when the target is a mountpoint or along the mountpoint.